soundio/soundio.h

```diff
@@ -18,14 +18,18 @@
 #define SOUNDIO_EXTERN_C
 #endif
 
-#if defined(_WIN32)
-#if defined(SOUNDIO_BUILDING_LIBRARY)
-#define SOUNDIO_EXPORT SOUNDIO_EXTERN_C __declspec(dllexport)
+#if defined(SOUNDIO_STATIC_LIBRARY)
```

Comment:
At the moment, I use it when building a static library on windows using msvc. I didn't touch the cmake files, but the define should be set there when building as a static library.